third_party/crypto/acme/acme.go

```diff
@@ -557,26 +560,102 @@ func (c *Client) HTTP01ChallengePath(token string) string {
 	return "/.well-known/acme-challenge/" + token
 }
 
+func (c *Client) encodeExternalAccountBinding(key crypto.Signer, url string, eab *ExternalAccountBinding) (string, string, []byte, error) {
+	phead := fmt.Sprintf(`{"alg":%q,"kid":%q,"url":%q}`, eab.KeyAlgorithm, eab.KID, url)
```

Comment:
🙄 why manually construct JSON?

Reply:
This is also done in `jws.go` :man_shrugging: